src/libraries/Microsoft.PowerFx.Core/Lexer/TexlLexer.cs

```diff
@@ -453,34 +193,124 @@ private void PopulateKeywordArrays()
             {
                 KeywordFalse, KeywordTrue, KeywordParent, KeywordSelf
             };
+        }
+
+        public static TexlLexer GetLocalizedInstance(CultureInfo culture)
+        {
+            culture ??= CultureInfo.InvariantCulture;
+            return culture.NumberFormat.NumberDecimalSeparator == PunctuatorDecimalSeparatorInvariant ? InvariantLexer : CommaDecimalSeparatorLexer;
+        }
+
+        public static IList<string> GetKeywordDictionary()
```

Comment:
>GetKeywordDictionary [](http://example.com/codeflow?start=36&length=20)

Is this called? (I don't see a caller) 

If so, can it just return _keywords.Values?  (avoid the array creation)

Reply:
used in PAClient... LanguageHelper